contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -739,7 +739,9 @@ def restart(self, timeout=10, clean=True):
         self.start()
 
     def fund_channel(self, l2, amount, wait_for_active=True, announce_channel=True):
```

Comment:
is there a way to mark this as deprecated?

Reply:
There's the `warnings` package which is a bit heavy handed (runtime warnings). I added a warning since this is mostly used in testing anyway.